db/dcrsqlite/sync.go

```diff
@@ -131,18 +131,24 @@ func (db *wiredDB) resyncDB(quit chan struct{}, blockGetter rpcutils.BlockGetter
 
 	// Attempt to rewind stake database, if needed, forcing it to the lowest DB
 	// height (or 0 if the lowest DB height is -1).
-	if stakeDBHeight > startHeight && stakeDBHeight > 0 {
-		if startHeight < 0 || stakeDBHeight > 2*startHeight {
+	if stakeDBHeight > dbHeight && stakeDBHeight > 0 {
+		if dbHeight < 0 || stakeDBHeight > 2*dbHeight {
 			return -1, fmt.Errorf("delete stake db (ffldb_stake) and try again")
 		}
-		log.Infof("Rewinding stake node from %d to %d", stakeDBHeight, startHeight)
-		// rewind best node in ticket db
-		stakeDBHeight, err = db.RewindStakeDB(startHeight, quit)
+		log.Infof("Rewinding stake node from %d to %d", stakeDBHeight, dbHeight)
+		// Rewind best node in ticket DB to larger of lowest DB height or zero.
+		stakeDBHeight, err = db.RewindStakeDB(dbHeight, quit)
 		if err != nil {
-			return startHeight, fmt.Errorf("RewindStakeDB failed: %v", err)
+			return dbHeight, fmt.Errorf("RewindStakeDB failed: %v", err)
 		}
 	}
 
+	// Start syncing at or after DB height depending on whether an external
+	// MasterBlockGetter is already configured to relay the current best block,
+	// in which case we receive and discard it to maintain synchronization with
+	// the auxiliary DB.
+	startHeight := dbHeight
```

Comment:
```golang
dbHeight, summaryHeight, stakeInfoHeight, stakeDBHeight, err := db.DBHeights()
```
Apart from change in name, is possible for `startHeight` to be assigned a different value than was originally assigned to `dbHeight` in the code snippet above?